geometry_msgs/Point position
geometry_msgs/WrenchStamped ee_force
bool open_grasp
bool done
float32 reward
